modin/pandas/test/test_dataframe.py

```diff
@@ -1868,8 +1868,13 @@ def test_mask(modin_df, pandas_df):
     ids=arg_keys("numeric_only", bool_none_arg_keys),
 )
 def test_max(request, modin_df, pandas_df, axis, skipna, numeric_only):
+    try:
+        pandas_result = pandas_df.max(axis=axis, skipna=skipna, numeric_only=numeric_only)
+    except:
+        with pytest.raises(TypeError):
+            modin_result = modin_df.max(axis=axis, skipna=skipna, numeric_only=numeric_only)
+        return
     modin_result = modin_df.max(axis=axis, skipna=skipna, numeric_only=numeric_only)
```

Comment:
can this part be moved to the try block so we wont have an empty return statement?

Reply:
The return is so that execution doesn't move to the next line (which raises an error without being caught, causing the test to fail). Moving the return statement to the try doesn't prevent this so the tests will still show up as failed.

Reply:
Not the return statement but the stuff below the return statement into the try block?

Reply:
ah yeah you're right - updated the PR!